packages/block-library/src/navigation/edit/index.js

```diff
@@ -342,20 +345,28 @@ function Navigation( {
 	const [ detectedOverlayColor, setDetectedOverlayColor ] = useState();
 
 	const onSelectClassicMenu = async ( classicMenu ) => {
-		const navMenu = await convertClassicMenu(
-			classicMenu.id,
-			classicMenu.name,
-			'draft'
-		);
-		if ( navMenu ) {
-			handleUpdateMenu( navMenu.id, {
-				focusNavigationBlock: true,
-			} );
-		}
+		return new Promise( async ( resolve ) => {
+			const navMenu = await convertClassicMenu(
+				classicMenu.id,
+				classicMenu.name,
+				'draft'
+			);
+			if ( navMenu ) {
+				handleUpdateMenu( navMenu.id, {
+					focusNavigationBlock: true,
+				} );
+			}
+			resolve();
+		} );
 	};
 
 	const onSelectNavigationMenu = ( menuId ) => {
-		handleUpdateMenu( menuId );
+		return new Promise( ( resolve ) => {
+			handleUpdateMenu( menuId, {
+				focusNavigationBlock: true,
+			} );
+			resolve();
+		} );
```

Comment:
Why are we using a `Promise` here when there are no async actions occuring?

Reply:
`handleUpdateMenu` is sync right?

Reply:
it's not needed indeed, it just liked the exaggerated consistency 😂